### PR TITLE
Added a VideoPage.py file which contains a VideoPage class

### LINKED ISSUE #2: Add class VideoPage in VideoPage.py
Like [PhotoPage class](https://github.com/dinhanhx/PerceiveYourMeme/blob/master/DOCUMENTATION.md#photopage), a VideoPage object should have

A VideoPage object has:
  - Attributes
    - `basic_info_dict` : a dictionary of keys (Video url, Id, Name)

  - Methods
    - `pprint()` : Pretty print of `basic_info_dict`

Example url https://knowyourmeme.com/videos/225020-western-animation

### PR DESCRIPTION
**Pull request by Rolv-Apneseth**
Done in relation to issue #2 

Is this ok or is there anything else you wanted in it to start off with.

### CODE REVIEW

File: PerceiveYourMeme/VideoPage.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+
+def isValid(url):
+    if url.startswith('https://knowyourmeme.com/videos/'):
+        http = urllib3.PoolManager()
+        response = http.request('GET', url, headers=HEADERS)
+        if response.status == 200:
+            return True
+        else:
+            return False
```
Comment (dinhanhx) [inline PerceiveYourMeme/VideoPage.py]: Can you make this shorter?

File: PerceiveYourMeme/VideoPage.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        # Contains all basic information about the video
+        self.basic_info_dict = {}
+
+        if isValid(url):
+            self.basic_info_dict['Video url'] = url
+
+            # Get name and id of video from url
+            id_name = url.split('/')[-1].split('-')
+            self.basic_info_dict['Id'] = id_name[0]
+            self.basic_info_dict['Name'] = ' '.join(id_name[1:])
```
Comment (dinhanhx) [inline PerceiveYourMeme/VideoPage.py]: Can you also get the youtube url from the KYM video page and store original url?

I expect sth like this
```
self.basic_info_dict['Youtube url'] = sth
self.basic_info_dict['Org url'] = url
```

**Comment by dinhanhx** [comment]
@Rolv-Apneseth Can you change/update the part I commented?

**Comment by Rolv-Apneseth** [comment]
So sorry, very new to contributing to projects. How do I see your comments?

**Comment by Rolv-Apneseth** [comment]
Also, not sure if it is important or not but my vscode gives me a warning for the `from .CONST import HEADERS, DEFAULT_DOWNLOAD_PATH` line in both my code and the PhotoPage.py saying "Attempted relative import beyond top-level package"

**Comment by dinhanhx** [comment]
@Rolv-Apneseth You go to this pull request, click `[Files changed](https://github.com/dinhanhx/PerceiveYourMeme/pull/11/files)` then you scroll down a little bit. You now should see some comments of mine. 

For vs code warning, it's normal to encounter these kind when working with Python package.

I suggest you make a Python script in this [folder](https://github.com/dinhanhx/PerceiveYourMeme/tree/master/PerceiveYourMeme) but do not add that file to git system.

Example

drive.py
```Python
import sys
sys.path.append('../')

from PerceiveYourMeme import PhotoPage, MemePage, NewsPage

p = PhotoPage('https://knowyourmeme.com/photos/1903951-2020-us-presidential-debate')
p.pprint()
p.download_photo()

p = MemePage('https://knowyourmeme.com/memes/crying-cat')
p.pprint()
p.download_origin_image()

p = NewsPage('https://news.knowyourmeme.com/news/mia-khalifa-is-auctioning-iconic-porn-glasses-to-raise-money-for-beirut')
p.pprint()
p.download_head_img()

```

**Comment by Rolv-Apneseth** [comment]
I'm afraid I can't see anything but what I wrote on that page. Is it an issue on my end?

**Comment by dinhanhx** [comment]
image
@Rolv-Apneseth Can you see these comments now?

**Comment by dinhanhx** [comment]
@Rolv-Apneseth Oh I am very sorry mate. Turn out, I never have presses the submit button.

**Comment by Rolv-Apneseth** [comment]
Hey, I've been trying to get the direct url for the video but I'm still learning html. The problem is the actual video link is in what looks like another html document embedded into the page's html? I can't seem to be able to get results with soup.find for stuff inside that file. Any suggestions?

**Comment by dinhanhx** [comment]
I suggest you parse the inside html document with BeautifulSoup() object again.

If that doesn't work for you, I will still merge your PR.

**Comment by Rolv-Apneseth** [comment]
Yeah sorry I can't figure it out, so I removed what I had tried there. Maybe you can raise another issue on it? Also, would you want me to try tidy up the PhotoPage.py or is it fine as is for you?

**Comment by Rolv-Apneseth** [comment]
Oh also if you don't mind, and if you are actually gonna merge this, could you add the hacktoberfest-accepted tag to the pr?

**Comment by Rolv-Apneseth** [comment]
I may as well leave this here too but this is where the other document is embedded on the page and I can't figure out how to parse it 
```<iframe class="embedly-embed" src="//cdn.embedly.com/widgets/media.html?src=https%3A%2F%2Fwww.youtube.com%2Fembed%2FKHhe8begOQk%3Ffeature%3Doembed&amp;url=https%3A%2F%2Fwww.youtube.com%2Fwatch%3Fv%3DKHhe8begOQk&amp;image=https%3A%2F%2Fi.ytimg.com%2Fvi%2FKHhe8begOQk%2Fhqdefault.jpg&amp;key=9a2135a1b52444f0b0bc53bbd4e50c36&amp;type=text%2Fhtml&amp;schema=youtube" scrolling="no" allowfullscreen="" width="640" height="360" frameborder="0"></iframe>```

**Comment by dinhanhx** [comment]
@Rolv-Apneseth Now you should see 1/4 in your hacktoberfest profile

**Pull request merged**